ui/selectors/confirm-transaction.js

```diff
@@ -222,7 +223,12 @@ export const sendTokenTokenAmountAndToAddressSelector = createSelector(
 export const contractExchangeRateSelector = createSelector(
   contractExchangeRatesSelector,
   tokenAddressSelector,
-  (contractExchangeRates, tokenAddress) => contractExchangeRates[tokenAddress],
+  (contractExchangeRates, tokenAddress) =>
+    contractExchangeRates[
+      Object.keys(contractExchangeRates).find((address) =>
```

Comment:
Yes I believe so. If we try to access the object with a key of `undefined` it will just return undefined. Which, if the `tokenAddress` doesn't exist in the `contractExchangeRates` object, will result in the same returned value `undefined` as before this change